### DISCUSSION TITLE
Fan Hubs

### DISCUSSION THREAD

**Discussion by CovertRockhound** [Rem0o/FanControl.Releases#688] [Ideas]
I am forced to use a fan hub due to a limited number of fan headers on my motherboard, so I picked up a Corsair Commander Pro. Its not compatible with FanControl, as one would expect (is ANYTHING compatible with Corsair products? lol) I've been using iCUE to control my fans and.... well.... its AWFUL. 

Anyway, I did a little googling and found a guy who has designed some very simplistic software that is able to control the Commander Pro. He seems to be quite happy to allow other people to use his code any way they see fit as discussed in the description of this video: https://www.youtube.com/watch?v=3VGf_aRou4s

So I thought I would bring his software to everybody's attention here, so that perhaps what he made could be integrated into FanControl to make it compatible with at least one fan hub. 

I know VERY little about coding, so I don't know if what I have proposed would be a huge headache, but I thought it may as well be brought up, since I'm sure there's a number of people like me that have to use a corsair fan hub to operate their fans. 

Thanks for reading my rambling. Here's a link to the software this guy made (he called it pCUE) https://github.com/crmaris/pCUE

**Comment by Rem0o** [comment]
https://github.com/iJacks1980/FanControl.CommanderPRO

**Reply by CovertRockhound** [reply]
Oh excellent! I'll give it a try. Thank you!

**Reply by CovertRockhound** [reply]
I've dropped the FanControl.CommanderPro.dll file into the Plugins folder and restarted the program, but when i start assisted setup and the sensor settings page opens, under the plugins section it says "no plugin loaded"

**Reply by CovertRockhound** [reply]
iJacks1980 said:

"in case anybody stumbles upon this the file was blocked by Windows by default - and can only be unblocked by going through a right click, properties, check the unblock this file box and pressing ok."

I followed these instructions and it resolved the issue.
